### PR TITLE
Feature/graph view

### PR DESCRIPTION
**Pull request by damien-nd**
- See if we need more options in MUGraphView.swift.

- Sorry for the documentation's files.

- We have to update to swift 5 for Sample.app + Playground.playground.

### CODE REVIEW

File: Sejima/Source/MUGraphView/CGContext+GraphLines.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            // Skip the center line.
+            guard index != 0 else { continue }
+
+            let position = floor(gridLineSpacing * CGFloat(index))
+            move(to: CGPoint(x: 0.0, y: position))
+            addLine(to: CGPoint(x: size.width, y: position))
+        }
+
+        // Stroke the lines.
+        setStrokeColor(UIColor.darkGray.cgColor)
```
Comment (MoveUpwardsDev) [inline Sejima/Source/MUGraphView/CGContext+GraphLines.swift]: Pas customizable ?

File: Sejima/Source/MUGraphView/CGContext+GraphLines.swift [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        strokePath()
+
+        // Add and stroke the center line.
+        move(to: .zero)
+        addLine(to: CGPoint(x: size.width, y: 0.0))
+
+        setStrokeColor(UIColor.lightGray.cgColor)
```
Comment (MoveUpwardsDev) [inline Sejima/Source/MUGraphView/CGContext+GraphLines.swift]: Pas customizable ?

File: Sejima/Source/MUGraphView/GraphSegment.swift [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+import UIKit
+import simd
+
+/// Apple's GraphSegment
+internal final class GraphSegment: UIView {
+
+    // MARK: - Properties
+
+    /// All values for this segments.
+    private(set) var dataPoints = [double3]()
```
Comment (MoveUpwardsDev) [inline Sejima/Source/MUGraphView/GraphSegment.swift:18]: We shall have a type alias to avoid double3 type

Comment (damien-nd) [reply Sejima/Source/MUGraphView/GraphSegment.swift:18]: MUGraphView doesn't import simd, but still in Apple's code in GraphView and GraphSegment.

**Pull request merged**